Formula/traildb.rb

```diff
@@ -0,0 +1,21 @@
+
+class Traildb < Formula
+  desc "Blazingly-fast database for log-structured data"
+  homepage "http://traildb.io"
+  url "https://github.com/traildb/traildb/archive/0.5.tar.gz"
+  sha256 "4d1b61cc7068ec3313fe6322fc366a996c9d357dd3edf667dd33f0ab2c103271"
+
+  depends_on "libarchive"
+  depends_on "homebrew/boneyard/judy"
```

Comment:
If `judy` is essentially done and dead by now, then vendoring it into this formula sounds like the most practical solution to me. (Of course having a maintained `judy` project would be even nicer. While the data structure and algorithms may be done, software never is as it always has to adapt to changes in compilers and environments, however small those changes might be if something is implemented in a safe subset of a pretty stable and standardized language.)